python/Scripts/baketextures.py

```diff
@@ -52,7 +55,13 @@ def main():
         print(msg)
 
     baseType = mx_render.BaseType.FLOAT if opts.hdr else mx_render.BaseType.UINT8
-    baker = mx_render_glsl.TextureBaker.create(opts.width, opts.height, baseType)
+    
+    
+    if platform == "darwin":
```

Comment:
I'm not sure if you want to force using the MSL baker on Mac as you cannot get baking with the GLSL baker anymore and can change CI results as this is run as part of the github action workflows. I assume it's possible to get different results purely based on platform ? 

I'm not sure what the option would be however. Perhaps an option `target` argument as with other scripts, and have it default to GLSL for now. 
@jstone-lucasfilm, what do you think ?

Reply:
That makes sense. Thanks for the comment. I addressed it by adding option to run using GLSL texture baker; default is still left as Metal.